package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "massa-unit-tests-src",
-  "lockfileVersion": 3,
+  "lockfileVersion": 2,
```

Comment:
Please update your npm version to have lock file updated and avoid pushing a new one with old version.

Reply:
I'm still creating lockfileVersion 2 with node 16 and npm 8.19.2 so I think we should keep it like that for now?

Reply:
Following this : https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json v3 shouldn't be used in package-lock at root. I let you change this then.